install/windows/CUsersoschi/.vagrant.d/Vagrantfile

```diff
@@ -0,0 +1,9 @@
+# These are requirements for this base Vagrantfile. If they are not
```

Comment:
The file name contains an unrecognized symbol `install/windows/CUsersoschi/.vagrant.d/Vagrantfile `. It might be not a problem.

Do we actually need this file or it was accidentally committed?